### DISCUSSION TITLE
实在搞不明白怎么配置dns

### DISCUSSION THREAD

**Discussion by scp096** [Openwrt-Passwall/openwrt-passwall2#713] [Q&A]
openwrt内访问google没问题，开启客户端代理；电脑设置网关指向openwrt，打开国内网站都没问题，识别ip也能识别出节点ip，但是打开google提示不是私密连接
![image](https://github.com/user-attachments/assets/b96f6e56-e306-46f5-9132-12ff568cede7)

**Comment by handsomeji** [comment]
網關、DNS伺服器都需要指向 openwrt，並且請關閉瀏覽器的內置DNS（DoH）（或叫安全DNS）

**Comment by LeoYanlv** [comment]
到高级设置 - Xray/Sing-Box 设置把覆盖连接目标地址
